### PR TITLE
Added execution order document

### PR DESCRIPTION
**Pull request by NGovani**
# Summary

Added a document which goes through the order of function calls during a turn of simulation including the functions called on agents.

IIGO not added yet due to forthcoming changes

## Test Plan

- No test needed

### CODE REVIEW

**Comment by QFSW** [comment]
# ⚠️Warning! This diff contains changes in protected folders

File: ExecutionOrder.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# SOMAS2020 Simulation Execution Order
```
Comment (lhl2617) [inline ExecutionOrder.md]: move to `docs/` and name it `EXECUTION_ORDER.md`? also add a reference to it on the main readme.

Comment (NGovani) [reply ExecutionOrder.md]: On it boss

**Review by lhl2617 — APPROVED** [review]
pre approving m8

**Pull request merged**